src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+
+  @for $starCount from 1 to 5 {
+    &--#{$starCount} &__star:nth-child(-n+#{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  & &__star:nth-child(5) {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
what does this part do?
what if i want `class="stars stars--5"` - i expect it to show 5 active stars.
It does not show me 5 stars (((

Reply:
i fixed it.